diffengine/stores/mongo.py

```diff
@@ -11,18 +11,18 @@
 logger = logging.getLogger("diffengine.stores.mongo")
 
 class Mongo(Store):
-    
+
     def __init__(self, db):
         self.db = db
         self.revisions = Revisions(self)
         self.engine_status = EngineStatus(self)
         self.processor_status = ProcessorStatus(self)
-    
+
     @classmethod
-    def from_params(cls, *args, db_name, **kwargs):
+    def from_params(cls, db_name, *args, **kwargs):
```

Comment:
In python3, this is a way to make keyword arguments required. 

    >>> def foo(*args, bar, **kwargs):
    ...     pass
    ... 
    >>> foo("test")
    Traceback (most recent call last):
      File "<stdin>", line 1, in <module>
    TypeError: foo() missing 1 required keyword-only argument: 'bar'
    >>> foo(bar="test")